src/overlays/actors/ovl_Bg_F40_Swlift/z_bg_f40_swlift.c

```diff
@@ -5,18 +5,21 @@
  */
 
 #include "z_bg_f40_swlift.h"
+#include "objects/object_f40_obj/object_f40_obj.h"
 
 #define FLAGS (ACTOR_FLAG_10)
 
 #define THIS ((BgF40Swlift*)thisx)
 
 void BgF40Swlift_Init(Actor* thisx, PlayState* play);
 void BgF40Swlift_Destroy(Actor* thisx, PlayState* play);
-void BgF40Swlift_Update(Actor* thisx, PlayState* play);
+void BgF40Swlift_Update(Actor* thisx, PlayState* play2);
 void BgF40Swlift_Draw(Actor* thisx, PlayState* play);
 
-#if 0
-ActorInit Bg_F40_Swlift_InitVars = {
+static s32 sParams[4] = { 0xFF, 0xFF, 0xFF, 0xFF };
+static s32 sHeights[4];
```

Comment:
```suggestion
static f32 sHeights[4];
```
Looks like `sHeights` is always used as an `f32`. Can you just type it as an array of `f32` instead of casting?

Reply:
It strangely needs the cast.

Reply:
Interesting. Can you leave this as an `s32`, and remove the `f32` casts in the code? Looks like all operations are with `f32` so they should implicitly cast.